packages/esm-patient-appointments-app/src/appointments/appointments.resource.ts

```diff
@@ -63,57 +58,7 @@ export function useAppointments(patientUuid: string, startDate: string, abortCon
   };
 }
 
-export function useAppointmentService() {
```

Comment:
All this stuff and the files deleted above are files that were previously copied to esm-appointments-app